app/Http/Controllers/ObjectivesController.php

```diff
@@ -227,9 +229,8 @@ public function update(Mission $missionAlias, Request $request, $slug, $mission_
                 */
                 $followedBy = Auth::user()
                                 ->FollowedBy;
-
                 foreach ($followedBy as $followedByKey => $followedByValue) {
-                  event(new ObjectiveComplete(['user_id' => Auth::user()->id, 'skill_name' => $skill->skill_name, 'post_content' => $objective->name, 'followed_id' => $followedByValue->user_id], Auth::user()));
+                  event(new ObjectiveComplete(['user_id' => Auth::user()->id, 'skill_name' => $skill->skill_name, 'post_content' => $objective->name, 'followed_id' => $followedByValue->user_id, 'post_id' => $createdPost->id], Auth::user()));
```

Comment:
Send the new posts ID as well so that can get mapped in the recipients chat pane